lib/utilities.js

```diff
@@ -26,6 +26,18 @@ async function getOSVersion (udid, socket) {
   }
 }
 
+async function getDeviceTime (udid, socket) {
+  const lockdown = await startLockdownSession(udid, socket);
+  try {
+    const epochValue = await lockdown.getValue({ Key: 'TimeIntervalSince1970' });
```

Comment:
should `TimeIntervalSince1970` be a constant?

Reply:
That makes sense. Should I put that in lockdown or utilities? I will put it to `utilities` for now